cluster-autoscaler/core/scaledown/actuation/update_latency_tracker.go

```diff
@@ -0,0 +1,107 @@
+package actuation
+
+import (
+	"time"
+
+	"k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes"
+	"k8s.io/autoscaler/cluster-autoscaler/utils/taints"
+	"k8s.io/klog/v2"
+)
+
+const sleepDurationWhenPolling = 50 * time.Millisecond
+const waitForTaintingTimeoutDuration = 2 * time.Minute
+
+type NodeTaintStartTime struct {
+	nodeName  string
+	startTime time.Time
+}
+
+type UpdateLatencyTracker struct {
```

Comment:
Please add a comment explaining the purpose of this struct.

Reply:
Done